src/main/java/evm/ContractTransaction.java

```diff
@@ -0,0 +1,138 @@
+package evm;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.client.*;
+import org.ethereum.vm.util.ByteArrayUtil;
+import org.ethereum.vm.util.HashUtil;
+import org.ethereum.vm.util.HexUtil;
+import static org.ethereum.vm.util.ByteArrayUtil.merge;
+import org.apache.log4j.Logger;
+
+public class ContractTransaction extends vmbase {
```

Comment:
missing java doc for the class.

Reply:
this class also needs supports of unit testing for its methods.

Reply:
All the functions in the class are automatically called by the **LightChainCV** class during its unit testing.

Reply:
> All the functions in the class are automatically called by the LightChainCV class during its unit testing.
That is an integration test, not a unit test. For unit testing, a unit of program needs to be tested in isolation.

Reply:
@KedarKshatriya we cannot merge a class that is not supported by unit testing.

Reply:
Added, check here db20b90eb21e9a4a55662adbd4a69ac06cd5d83a